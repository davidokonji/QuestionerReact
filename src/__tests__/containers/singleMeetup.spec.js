import { shallow } from 'enzyme';
import React from 'react';
import { SingleMeetup } from '../../containers/SingleMeetup.container';

describe('Single Meetup container', () => {
  it('should  render Single Meetup container with props', () => {
    const singleMeetup = jest.fn();
    const match = {
      params: {
        id: 1
      }
    };
    const wrapper = shallow(<SingleMeetup singleMeetup={singleMeetup} match={match} />);

    wrapper.setState({
      open: false
    });

    expect(wrapper).toMatchSnapshot();
  });
});
