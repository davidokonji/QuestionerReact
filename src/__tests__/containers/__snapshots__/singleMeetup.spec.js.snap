// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Single Meetup container should  render Single Meetup container with props 1`] = `
<div>
  <ConnectFunction />
  <div
    className="container mt-5 shadow-sm"
  >
    <ToastContainer
      autoClose={1000}
      bodyClassName={null}
      className={null}
      closeButton={false}
      closeOnClick={true}
      draggable={true}
      draggablePercent={80}
      hideProgressBar={true}
      newestOnTop={false}
      pauseOnFocusLoss={true}
      pauseOnHover={true}
      position="top-right"
      progressClassName={null}
      progressStyle={null}
      rtl={false}
      style={null}
      toastClassName={null}
      transition={[Function]}
    />
    <SingleMeetupTop
      open={true}
      toggle={[Function]}
    />
    <SingleMeetupBottom />
  </div>
  <Footer />
</div>
`;

exports[`Single Meetup container should  render Single Meetup container with props 2`] = `
<div>
  <ConnectFunction />
  <div
    className="container mt-5 shadow-sm"
  >
    <ToastContainer
      autoClose={1000}
      bodyClassName={null}
      className={null}
      closeButton={false}
      closeOnClick={true}
      draggable={true}
      draggablePercent={80}
      hideProgressBar={true}
      newestOnTop={false}
      pauseOnFocusLoss={true}
      pauseOnHover={true}
      position="top-right"
      progressClassName={null}
      progressStyle={null}
      rtl={false}
      style={null}
      toastClassName={null}
      transition={[Function]}
    />
    <SingleMeetupTop
      open={true}
      toggle={[Function]}
    />
    <SingleMeetupBottom />
  </div>
  <Footer />
</div>
`;
